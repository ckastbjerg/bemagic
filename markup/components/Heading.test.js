/* eslint-env jest */
const Heading = require('./Heading');

test('returns a string of attributes from an object', () => {
    global.config = require('../../config');

    expect(Heading({
        backgroundClass: 'background',
        themes: [
            { name: 'secondary', classes: 'prefix-theme-secondary' },
            { name: 'inverse', classes: 'prefix-theme-inverse' },
        ],
        componentName: 'block',
    })).toMatchSnapshot();
});
