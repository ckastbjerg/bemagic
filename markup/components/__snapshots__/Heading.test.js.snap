// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns a string of attributes from an object 1`] = `
"
        <div class=\\"bemagic-page__heading bemagic-page-heading\\">
            block
            <div class=\\"bemagic-page-heading__explain\\">
                Styled for themes:
            </div>
            
        <div
            class=\\"bemagic-theme-button bemagic-theme-button--small prefix-theme-secondary background js-page-theme\\"
            data-section-ref=\\"block\\"
            data-theme=\\"secondary\\"
        >
        </div>
    
        <div
            class=\\"bemagic-theme-button bemagic-theme-button--small prefix-theme-inverse background js-page-theme\\"
            data-section-ref=\\"block\\"
            data-theme=\\"inverse\\"
        >
        </div>
    
        </div>
    "
`;
